BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Network considerations for Azure VMware Solution dual-region deployments with Secure vWAN
+description: Learn how to configure network connectivity when Azure VMware Solution private clouds are deployed in two Azure regions with Secure vWAN.
+author: jmedina
+ms.author: jmedina
+ms.date: 06/22/2023
+ms.topic: conceptual
+ms.service: caf
+ms.subservice: caf-scenario-vmware
+ms.custom: think-tank, e2e-azure-vmware
+---
+
+# Network considerations for Azure VMware Solution dual-region deployments using Secure vWAN with Routing-Intent
+
+This article describes the best practices for connectivity, traffic flows, and high availability of dual-region Azure VMWare Solution when using Azure vWAN with Routing Intent. This article will break down vWAN with Routing Intent topology from the perspective of AVS private clouds, On-premise sites, and Azure native. How to implement and configure Secure vWAN with Routing Intent is out of the scope and will not be discussed in this document.
+
+The document assumes readers have a basic understanding of vWAN and Secure vWAN with Routing Intent.
+
+**What is Azure Virtual WAN?**  
+[https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies](https://learn.microsoft.com/en-us/azure/virtual-wan/virtual-wan-about)
+
+**How to configure Virtual WAN Hub routing intent and routing policies**  
+https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies
```

Comment:
Check hyperlink syntax [link text](url)
Also, remove the "/en-us" fragment from the path to allow readers to browse to their localized content when clicking on the link (this is required for public doc content)

Reply:
@fguerri, I corrected this and made sure to remove "en-us" on all URL paths.